pkg/fleetautoscalers/fleetautoscalers.go

```diff
@@ -49,18 +52,29 @@ var client = http.Client{
 	},
 }
 
+// InactiveScheduleError denotes an error for schedules that are not currently active.
+type InactiveScheduleError struct{}
+
+func (m *InactiveScheduleError) Error() string {
```

Comment:
no reason to make this a pointer receiver, you can literally just do:

```
func (InactiveScheduleError) Error() string {
```

together with the `Is` it looks like: https://go.dev/play/p/u-QmYmCFfL_x